binder/start

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 
 # Replace DASK_DASHBOARD_URL with the proxy location
-sed -i -e "s|DASK_DASHBOARD_URL|${JUPYTERHUB_BASE_URL}user/${JUPYTERHUB_USER}/proxy/8787|g" binder/jupyterlab-workspace.json
+sed -i -e "s|DASK_DASHBOARD_URL|${JUPYTERHUB_BASE_URL}user/${JUPYTERHUB_USER}/proxy/8787|g" binder/example.jupyterlab-workspace
 
+# TODO: Importing of workspace via CLI is currently broken.
+# remove this workaround once https://github.com/jupyterlab/jupyterlab/issues/10805 is resolved
+mkdir -p ~/.jupyter/lab/workspaces/
+mv binder/example.jupyterlab-workspace ~/.jupyter/lab/workspaces/default-37a8.jupyterlab-workspace
```

Comment:
The `37a8` part of this path seems quite specific. Why is that part needed? 

```suggestion
mv binder/example.jupyterlab-workspace ~/.jupyter/lab/workspaces/default-37a8.jupyterlab-workspace
```

Reply:
This is 100% a hack -- that's a hash based on the workspace name which is not guaranteed to be stable. But I think it might be okay for the time-being, and is better than the currently broken state.

Reply:
Gotcha -- totally fine with me if it works